### ISSUE TITLE
Badge component minimum color contrast ratio

### ISSUE THREAD

**Issue by canerakdas** [nodejs/nodejs.org#8464]
For `Badge` component, the contrast ratio of the `neutral` kind does not meet our minimum contrast standards.

<img width="360" alt="Image" src="https://github.com/user-attachments/assets/ca841336-dbb4-4dc2-97f3-0a256b183451" />

Ref: https://dequeuniversity.com/rules/axe/4.11/color-contrast
Example usage: https://nodejs.org/en/about/previous-releases

**Comment by malav2110** [comment]
Working on it.

**Comment by malav2110** [comment]
@canerakdas I believe bumping the neutral color from `bg-neutral-600` to `bg-neutral-800` will fix the issue. Thoughts?

**Comment by malav2110** [comment]
I am also noticing the color contrast issue in the `Node.js Releases` label. Do you want me to fix it in the same PR or you want me to create a separate issue and PR?

<img width="1005" height="574" alt="Image" src="https://github.com/user-attachments/assets/28b702b3-56ff-4957-8d5d-c480dc21bb26" />

**Comment by canerakdas** [comment]
> I am also noticing the color contrast issue in the `Node.js Releases` label. Do you want me to fix it in the same PR or you want me to create a separate issue and PR?
> 
> <img alt="Image" width="1005" height="574" src="https://private-user-images.githubusercontent.com/23551157/530860242-28b702b3-56ff-4957-8d5d-c480dc21bb26.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjcxMDUxMjIsIm5iZiI6MTc2NzEwNDgyMiwicGF0aCI6Ii8yMzU1MTE1Ny81MzA4NjAyNDItMjhiNzAyYjMtNTZmZi00OTU3LThkNWQtYzQ4MGRjMjFiYjI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjMwVDE0MjcwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2NWYxYWE4ZmVlYzdiYjUwOTc4YmQwMzhlMmI3Y2IxNmJhMjkyMWIyNzVlMzE5M2EwYWM2ZDc5YTZkODlhZWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3lUfDeHpLwYBqATUq8-4NwUyQaF0jOAcVmRtYIMBovE">

Thanks for pointing it out ✨ I think we can move this forward within the same PR 🙇

**Comment by malav2110** [comment]
@canerakdas bumped up `Node.js Releases` associated label color from `bg-green-600` to `bg-green-800`.
